src/py/flwr/simulation/ray_transport/ray_client_proxy.py

```diff
@@ -29,7 +29,7 @@
 class RayClientProxy(ClientProxy):
     """Flower client proxy which delegates work using Ray."""
 
-    def __init__(self, client_fn: ClientFn, cid: str, resources: Dict[str, int]):
+    def __init__(self, client_fn: ClientFn, cid: str, resources: Dict[str, Any]):
```

Comment:
The type of `resources` needs to be the same as the type of `client_resources` in `src/py/flwr/simulation/app.py`.